### ISSUE TITLE
/opt/conda/lib/python3.7/site-packages/sklearn/svm/base.py:929: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.   "the number of iterations.", ConvergenceWarning)

### ISSUE THREAD

**Issue by 123rugby** [hdcouture/TOCCA#1]
the problem :"/opt/conda/lib/python3.7/site-packages/sklearn/svm/base.py:929: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.   "the number of iterations.", ConvergenceWarning)"  always appeared and it retraining again and again

![image](https://user-images.githubusercontent.com/24840860/103450283-cec6e200-4cee-11eb-8e89-b346cf8665e5.png)

**Comment by 123rugby** [comment]
None
Train on 10000 samples, validate on 50000 samples
Epoch 1/100
 - 8s - loss: 719.7862 - softmax_loss: 2.5178 - concatenate_5_loss: 634.5027 - softmax_acc: 0.1393 - softmax_acc_1: 0.1345 - concatenate_5_acc: 0.0118 - val_loss: 1946347486748.5203 - val_softmax_loss: 2.7855 - val_concatenate_5_loss: 1478805920840.8899 - val_softmax_acc: 0.1099 - val_softmax_acc_1: 0.1511 - val_concatenate_5_acc: 5.6000e-04
·············
Epoch 99/100
 - 4s - loss: 1264.9329 - softmax_loss: 2.2491 - concatenate_4_loss: 873.4405 - softmax_acc: 0.1541 - softmax_acc_1: 0.1542 - concatenate_4_acc: 0.0087 - val_loss: 17434851524.0051 - val_softmax_loss: 2.2501 - val_concatenate_4_loss: 13191326079.6346 - val_softmax_acc: 0.1657 - val_softmax_acc_1: 0.1651 - val_concatenate_4_acc: 0.0109
Epoch 100/100
 - 4s - loss: 1211.0534 - softmax_loss: 2.2484 - concatenate_4_loss: 839.0251 - softmax_acc: 0.1552 - softmax_acc_1: 0.1517 - concatenate_4_acc: 0.0095 - val_loss: 15156362767.5705 - val_softmax_loss: 2.2586 - val_concatenate_4_loss: 11481265112.6588 - val_softmax_acc: 0.1856 - val_softmax_acc_1: 0.1091 - val_concatenate_4_acc: 0.0016
Fitting 5 folds for each of 15 candidates, totalling 75 fits
[Parallel(n_jobs=4)]: Using backend LokyBackend with 4 concurrent workers.
/opt/conda/lib/python3.7/site-packages/sklearn/svm/base.py:929: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.   "the number of iterations.", ConvergenceWarning)
/opt/conda/lib/python3.7/site-packages/sklearn/svm/base.py:929: ConvergenceWarning: Liblinear failed to converge, increase the number of iterations.   "the number of iterations.", ConvergenceWarning)
[Parallel(n_jobs=4)]: Done  75 out of  75 | elapsed:  7.7min finished
train 0.118100 val 0.117660 test 0.121900
Fold 5 of 5
2021-01-02 11:31:11.252437: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2021-01-02 11:31:11.252531: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-01-02 11:31:11.252548: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2021-01-02 11:31:11.252564: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2021-01-02 11:31:11.252674: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 3251 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-32GB, pci bus id: 0000:b2:00.0, compute capability: 7.0)
_
